tests/Architecture/ActionsTest.php

```diff
@@ -0,0 +1,7 @@
+<?php
+
+test('actions test')
+    ->expect('Cachet\Actions')
+    ->toBeClasses()
+    ->toExtendNothing()
+    ->toHaveMethod('handle');
```

Comment:
I think we should also check that Actions use the `AsAction` trait for consistency.